manifests/config.pp

```diff
@@ -33,4 +33,12 @@
     mode    => '0640',
     content => $qpid::acl_content,
   }
+
+  file { '/var/lib/qpidd':
```

Comment:
I've tried to avoid hardcoding any paths and still think that's a good pattern to keep. Experience tells me that paths can change, either in SCL or in a new major OS version. That's why I'd prefer to keep it here too.